frontend/views/containers/payments/RecordPayment.vue

```diff
@@ -0,0 +1,252 @@
+<template lang='pug'>
+modal-base-template(ref='modal' :fullscreen='true' class='has-background')
+  .c-header(:class='{"hide-desktop": donePayment}')
+    i18n.is-title-2.c-title(tag='h2') Record payments
+
+  .c-payment-form(v-if='!donePayment')
+    .c-content
+      form.card.c-card(
+        @submit.prevent=''
+        novalidate='true'
+      )
+        i18n.has-text-bold.c-title(tag='h3') Who did you send money to?
+        payments-list(
+          :titles='tableTitles'
+          :payments='paymentsDistribution'
+          paymentsType='edit'
+        )
+
+        .c-footer
+          .c-footer-info
+            i18n.has-text-bold(
+              tag='h4'
+              :args='{ optional: `<span class="has-text-small has-text-1 has-text-normal">(${L("optional")})</span>`}'
+            ) Add a note {optional}
+
+            i18n.has-text-small.has-text-1(
+              tag='p'
+            ) Leave a message to the group members selected above.
+          .c-footer-action
+            input.switch#showComment(
+              type='checkbox'
+              name='displayComment'
+              @change='displayComment = !displayComment'
+            )
+            i18n.sr-only(tag='label' for='displayComment') Toggle comment box
+
+        transition(name='slidedown')
+          label.field(v-if='displayComment')
+            i18n.sr-only.label Leave a message
+            textarea.c-comment(
+              rows='4'
+            )
+
+        .buttons
+          i18n.is-outlined(
+            tag='button'
+            @click='closeModal'
+          ) Cancel
+
+          button.is-success(
+            @click='submit'
+            :disabled='this.recordNumber === 0'
+          ) {{ registerPaymentCopy }}
+
+  .c-payment-success(v-else)
+    svg-success
+    i18n.is-title-2.c-title(tag='h2') Your payments were recorded
+    banner-simple(severity='info')
+      i18n(
+        compile
+        :args='{ r1: `<a class="link" href="https://donorbox.org/okturtles-donation" target="_blank">`, r2: "</a>"}'
+      ) Please consider supporting the development of Group Income by {r1}sending a donation{r2}!
```

Comment:
A few things: here `compile` attribute isn't necessary because the HTML being used is not a Vue.js component but normal HTML.

We also don't want them to just consider donating, but to actually donate, so please update this sentence to read:

```
Please support the development of Group Income by {r1}sending a donation{r2}!
```

(I know the copy in the design is different, but let's just improve it now) cc @mmbotelho can you update the design docs as well?

Reply:
Done @taoeffect @pieer !